frontend/src/core/comments/components/AddComment.js

```diff
@@ -80,9 +79,26 @@ export default function AddComments(props: Props) {
                     rows={ minRows }
                     ref={ commentInput }
                     onChange={ handleOnChange }
-                    onKeyDown={ submitComment }
+                    onKeyDown={ e =>
+                        (e.keyCode === 13 && e.shiftKey === false) ? submitComment(e) : null
+                    }
```

Comment:
Please change this to `handleOnKeyDown` for consistency and clarity. I believe it will be easier to read.